src/drivers/github.go

```diff
@@ -126,7 +126,7 @@ func (d *githubCodeHostingDriver) mergePullRequest(options MergePullRequestOptio
 		return "", fmt.Errorf("cannot merge via Github since there is no pull request")
 	}
 	if options.LogRequests {
-		d.log("GitHub API: Merging PR #%d", options.PullRequestNumber)
+		d.log("GitHub API: Merging PR", options.PullRequestNumber)
```

Comment:
From the related issue,  i don't think this would match the expected output.

Expected: `GitHub API: Merging PR #56`
Logically what I think this will print: `GitHub API: Merging PR 56`

Reply:
You are right. Addressing in https://github.com/git-town/git-town/pull/1603